pkg/services/hivelocity/device/device.go

```diff
@@ -42,6 +42,7 @@ const (
 )
 
 // Service defines struct with machine scope to reconcile Hivelocity machines.
+// question: why is Service needed?
```

Comment:
Otherwise, we don't have a struct that gets the method "Reconcile". But this can be done differently for sure.

Reply:
ok.

Reply:
@janiskemper 
I suggest to remove the struct Service and put the code into MachineScope. But I am fine if you disagree, then I would just remove the comment.

Reply:
ok, I removed the question.